documentacion/como_empezar/README.md

```diff
@@ -208,7 +208,7 @@ valores entregados por el comercio al principio del flujo transaccional.
 
 2. Posterior a que Transbank confirme que la planilla de integración se encuentra correcta, se solicitará la [generación de las credenciales del comercio](#credenciales-en-onepay) (llave privada y certificado publico). El certificado público debe ser enviado junto al logo del comercio a soporte@transbank.cl para su registro.
 
-3. Transbank informará el correcto registro del certificado público del comercio. Posterior a ello, será necesario cambiar la configuración del e-commerce para funcionar en producción.
+3. Transbank informará el correcto registro del certificado público del comercio. Posterior a ello, será necesario [cambiar la configuración del e-commerce para funcionar en producción](#configuracion-para-produccion-utilizando-los-sdk)
```

Comment:
wouldn't be useful to add how are we going to contact them (mail, phone,sms) and how many days it will take?